barretenberg/cpp/src/barretenberg/sumcheck/instance/prover_instance.cpp

```diff
@@ -12,8 +12,8 @@ namespace bb {
  */
 template <class Flavor> size_t ProverInstance_<Flavor>::compute_dyadic_size(Circuit& circuit)
 {
-    // minimum circuit size due to lookup argument
-    const size_t min_size_due_to_lookups = circuit.get_tables_size() + circuit.get_lookups_size();
+    // for the lookup argument the circuit size must be at least as large as the sum of all tables used
+    const size_t min_size_due_to_lookups = circuit.get_tables_size();
```

Comment:
This was a holdover from plookup which required a polynomial that stored a concatenation of the tables and the lookup data. For log deriv lookups we only need a poly which stores the tables themselves.

Reply:
nice